Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18-bullseye
+# Install the dependencies for canvas
+RUN apt-get update
+RUN apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
```suggestion
# Install deps for canvas
RUN apt-get install build-essential libcairo2-dev libpango1.0-dev
```

Reply:
Are you sure we don't need those and cavas will work without them?

Reply:
Yes, taken from node-canvas's readme:
> Note that libgif/giflib, librsvg and libjpeg are optional and only required if you need GIF, SVG and JPEG support, respectively.